lib/test/sfTestFunctionalBase.class.php

```diff
@@ -478,7 +478,7 @@ static public function handlePhpError($errno, $errstr, $errfile, $errline)
   /**
    * Exception handler for the current test browser instance.
    *
-   * @param Exception $exception The exception
+   * @param Throwable $exception The exception
```

Comment:
Throwable was only available from php 7: https://www.php.net/manual/en/class.throwable.php

```suggestion
   * @param Throwable|Exception $exception The exception
```